### DISCUSSION TITLE
keybindings

### DISCUSSION THREAD

**Discussion by nexdep** [proginosko/LeechBlockNG#502] [Ideas]
would love some keybindings, at least for the most basic options:

- open settings
- start a lockdown periods
- etc

love, love, love this extension, keep the good work up!

**Comment by proginosko** [comment]
This should be easy to do. Stay tuned!

**Reply by nexdep** [reply]
awesome!

**Comment by proginosko** [comment]
Added in version 1.6.7.

**Reply by nexdep** [reply]
great :) how are they activated? I am in 1.6.7. and I cannot find in the options how to set them up..

**Reply by proginosko** [reply]
Go to about:addons, Extensions, click on the cog icon, select "Manage Extension Shortcuts".

**Reply by nexdep** [reply]
thanks, it was an easy question actually..
